code_challenges.rb

```diff
@@ -31,3 +42,27 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def is_palindrome = 'racecar'
+    if is_palindrome = is_palidrome.reverse
+        'racecar is a palidrome'
+    elsif
+        'racecar is not a palindrome'
+    end
+end
+
+def is_palindrome = 'LEARN'
+    if is_palindrome = is_palidrome.reverse
+        'LEARN is a palidrome'
+    elsif
+        'LEARN is not a palindrome'
+    end
+end
+
+def is_palindrome = 'rotator'
+    if is_palindrome = is_palidrome.reverse
+        'rotator is a palidrome'
+    elsif
+        'rotator is not a palindrome'
+    end
+end
```

Comment:
there is a concept in Development called D.R.Y. (Don't Repeat Yourself)-- it's one of the core principles in coding. 

Essentially its this: do not have multiple sets of code that does the same work if a more dynamic version of your code will do all the work. 

so if I were to refactor what you have it would look like this: 
```ruby
def is_palindrome word
    if word == word.reverse
        "#{word} is a palindrome"
    else
        "#{word} is not a palindrome"
    end
end


is_palindrome("racecar")
is_palindrome("LEARN")
is_palindrome("rotator")
```

Instead of copying and pasting  code three times I have made the code dynamic so that it can be called with any word any number of times.